src/api/services/configs.ts

```diff
@@ -82,13 +81,13 @@ export const getAllChanges = (
   let queryString = "";
   Object.keys(queryParams).forEach((key) => {
     if (queryParams[key] && queryParams[key] !== "All") {
-      const queryKey = key === "type" ? `config.config_type` : key;
+      const queryKey = key === "type" ? `config.type` : key;
       queryString += `&${queryKey}=eq.${queryParams[key]}`;
     }
   });
   return resolve(
     ConfigDB.get<ConfigTypeChanges[]>(
-      `/config_changes?order=created_at.desc${pagingParams}&select=*,config:config_names!inner(id,name,config_type)${queryString}`,
+      `/config_changes?order=created_at.desc${pagingParams}&select=*,config:config_names!inner(id,name,config_class)${queryString}`,
```

Comment:
```suggestion
      `/config_changes?order=created_at.desc${pagingParams}&select=*,config:config_names!inner(id,name,type)${queryString}`,
```

Reply:
It seems we need to update config_names as well to add type as well.